### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aleicazatti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
...
+## The Zen of PreliZ
+* Being open source. 
+* Avoid automated solutions.
+* Separate tasks among humans and computers.
+* Prevent overconfidence on users opinions.
+* Easily integrate with other tools.
+* Allow to perform prior predictive checks.
+* Having a simple and intuitive interface suitable for non-specialists in order to minimize cognitive biases and heuristics.
+* Switching between different types of visualization such as kernel density estimates plots, quantile dotplots, histograms, etc. 
+* Being agnostic of the underlying programming language, or at least interoperable with as many languages as possible.
```
Comment (aloctavodia) [inline README.md:16]:
```suggestion
* Being agnostic of the underlying probabilistic programming language
```
This is a little bit mote easy to achieve, although we can make this not just a Python project but also a Julia project as with main ArviZ, but that is a little bit more complex

File: README.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
...
+* Working on top of open source libraries related to Bayesian workflow and uncertainty visualization (ggdist, Bayesplot, ArviZ).
```
Comment (aloctavodia) [inline README.md:17]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
...
+* Modularity.
```
Comment (aloctavodia) [inline README.md:18]:
```suggestion
* Being modular
```

File: README.md [hunk 4]
```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
 
 **This package is very new and all of its features are experimental, not yet well tested, and subject to change without notice.**
 
+## The Zen of PreliZ
+* Being open source.
```
Comment (aloctavodia) [inline README.md:8]: Do we want to be open in other ways? like being also community-driven, being diverse?

File: README.md [hunk 5]
```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
 
 **This package is very new and all of its features are experimental, not yet well tested, and subject to change without notice.**
 
+## The Zen of PreliZ
+* Being open source. 
+* Avoid automated solutions.
+* Separate tasks among humans and computers.
```
Comment (aloctavodia) [inline README.md:10]: Maybe we need to be more clear here, maybe something like this. or something more explicit

```suggestion
* Separate tasks between humans and computers, from each according to his ability.
```

File: README.md (hunk 4 above)
Comment (aleicazatti) [inline README.md:8]:
```suggestion
* Being open source, community-driven, diverse and inclusive.
```

File: README.md (hunk 5 above)
Comment (aleicazatti) [reply README.md:10]:
```suggestion
* Separate tasks between humans and computers, from each according to his abilities. So numerical tasks that are computationally demanding, error-prone or even tedious should be automatized as much as possible, while allowing the user to retain control of important decisions.
```

Comment (aloctavodia) [reply README.md:10]:
```suggestion
* Separate tasks between humans and computers, so users can retain control of important decisions while numerically demanding, error-prone or tedious tasks are automatized.
```

**Review by aloctavodia — APPROVED** [review]

**Pull request merged**